cmd/experimental/kjobctl/apis/v1alpha1/application_profile_types.go

```diff
@@ -93,7 +93,7 @@ type TemplateReference string
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('ntasks' in self.requiredFlags) || self.name == 'Slurm'", message="ntasks flag can be used only on Slurm mode"
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('output' in self.requiredFlags) || self.name == 'Slurm'", message="output flag can be used only on Slurm mode"
 // +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('partition' in self.requiredFlags) || self.name == 'Slurm'", message="partition flag can be used only on Slurm mode"
-// +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('priority' in self.requiredFlags) || self.name == 'Slurm'", message="priority flag can be used only on Slurm mode"
+// +kubebuilder:validation:XValidation:rule="!has(self.requiredFlags) || !('priority' in self.requiredFlags) || self.name != 'Interactive'", message="priority flag can't be used on Interactive mode"
```

Comment:
Why interactive is not allowed?

Reply:
In interactive mode we are using plain pod to run interactive shell. As I can see on this table https://kueue.sigs.k8s.io/docs/overview/#job-integrated-features it's still not supported. 

Please tell me if I'm wrong.

Reply:
That was false assumption that pods don't allow to use
WorkloadPriorityClass.
We double checked with the team and it is not the case -  it will be fixed.
Thanks for spotting that!

Reply:
Fixed on docs https://github.com/kubernetes-sigs/kueue/pull/3216.